crypto/fipsmodule/rsa/padding.c

```diff
@@ -506,7 +506,7 @@ int RSA_verify_PKCS1_PSS_mgf1(const RSA *rsa, const uint8_t *mHash,
   //	-1	sLen == hLen
```

Comment:
What do you think about updating this comment and the one on 619? It's interesting because if we update the comment there is some implied info that RSA_PSS_SALTLEN_DIGEST is negative which shouldn't matter with a constant.

Reply:
Other SSL does not have this comment changed after adding the macro, and other cases (like `RSA_PSS_SALTLEN_MAX(-3)` and `RSA_PSS_SALTLEN_AUTO(-2)`) are not added yet. So I plan to keep existing comment.

Reply:
I think you should change it. It will remind you later on to change the comment when you add the other macros.

Reply:
Added the change. But I think this may not be necessary.
* `-1` is more straightforward (no need to search what the `RSA_PSS_SALTLEN_DIGEST` is).

```
// If |sLen| is RSA_PSS_SALTLEN_DIGEST then the salt length is the same as 
// the hash length. If -2, then the salt length is maximal given the space in |EM|.
//
```